frontend/src/components/ShootWorkers/MachineType.vue

```diff
@@ -121,14 +166,40 @@ export default {
     },
     hint () {
       return this.notInList ? 'This machine type may not be supported by your worker' : ''
+    },
+    cpuItems () {
+      const cpuItems = uniq(map(this.machineTypes, 'cpu'))
+      return ['all', ...cpuItems.sort((a, b) => (a - b))]
+    },
+    gpuItems () {
+      const gpuItems = uniq(map(this.machineTypes, 'gpu'))
+      return ['all', ...gpuItems.sort((a, b) => (a - b))]
+    },
+    memoryItems () {
+      const memoryItems = uniq(map(this.machineTypes, 'memory'))
+      return ['all', ...memoryItems.sort((a, b) => this.rawMemoryValue(a) - this.rawMemoryValue(b))]
     }
   },
   validations,
   methods: {
+    rawMemoryValue (memoryString) {
```

Comment:
- we should add tests for this function
- also support more units? https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory
- what if the unit is unknown/unsupported?